app/Actions/SSL/CheckSslExpiry.php

```
@@ -0,0 +1,94 @@
+     * domains, and optionally send the expiry notification.
+     *
+     * @throws ValidationException
+     */
+    public function check(Ssl $ssl, bool $notify = true, mixed $ssh = null): void
+    {
+        $server = $ssl->site?->server;
+
+        if ($server === null || $ssl->certificate_path === null) {
+            throw ValidationException::withMessages([
+                'ssl' => 'This certificate cannot be checked.',
+            ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Validation\ValidationException' in method 'check'. (undefined)

(StaticAccess)